policyengine_us/variables/gov/states/me/adjusted_gross_income/me_pension_income_subtractions.py

```diff
@@ -0,0 +1,54 @@
+from policyengine_us.model_api import *
+
+
+class me_pension_income_subtractions(Variable):
```

Comment:
Would you mind changing this variable name to `me_pension_income_deduction` to be consistent with the worksheet?